bhop.s

```diff
@@ -1158,6 +1161,14 @@ preserve_release_delay:
         jsr trigger_sample
         jsr queue_sample
 skip_sample_trigger:
+        lda channel_rstatus, x
+        and #ROW_HOLD_INSTRUMENT
+        beq init_instrument
+        lda channel_rstatus, x
+        and #($FF - ROW_TRIGGERED)
+        sta channel_rstatus, x
+        bne done_with_bytecode
+init_instrument:
```

Comment:
I'm not so sure about this part. Does any of this still need to be done even when holding instrument?